src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -1102,22 +1102,6 @@ public function testDumpKeepsExistingPermissionsWhenOverwritingAnExistingFile()
         $this->assertFilePermissions(745, $filename);
     }
 
-    /**
-     * @expectedException \Symfony\Component\Filesystem\Exception\IOException
-     * @expectedExceptionMessageRegExp /^Unable to detect if the target directory ".*" exists\.$/
-     */
-    public function testDumpFailsWithExceptionIfExecutablePermissionsForTheParentDirectoryAreMissing()
-    {
-        $this->markAsSkippedIfChmodIsMissing();
-
-        $target = $this->workspace.DIRECTORY_SEPARATOR.'foo';
-        $file = $target.DIRECTORY_SEPARATOR.'foobar';
-        mkdir($target);
-        chmod($this->workspace, 0666);
```

Comment:
For some reason, the filesystem on Travis CI does not care about the permissions at all. Even if I set it to `0` here, the code in `dumpFile()` is still able to access the directory.

Reply:
I don't think that this is a nice solution. But it will at least make tests pass again allowing us to rely on the build results again.